cypress/TestCases/FireboltCertification/UserGrants.md

```diff
@@ -0,0 +1,77 @@
+                                                    {
+                                                        "capability": "xrn:firebolt:capability:usergrant:acknowledgechallenge"
+                                                    }
+                                                ]
+                                            }
+                                        ],
+                                        "scope": "app",
+                                        "lifespan": "once",
+                                        "overridable": false
+                                    }
+                           ```         
+
+| Field         | Type    | Description                                                                                                                                             |
+| ------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| steps         | Array   | An array of GrantSteps, all of which need to be verified to fulfill the GrantPolicy.                                                                    |
+| capability    | String  | Refers to a specific granting capability.                                                                                                               |
+| configuration | Object  | Contains configuration details on how the capability should be set up or used..                                                                         |
+| scope         | String  | Defines the scope of the grant, specifying whether it applies to all apps on the device or just the requesting app. Possible values: `app` or `device`. |
+| lifespan      | String  | Indicates the duration for which the grant is valid. Possible values: `once`, `forever`, `seconds`, `powerActive`, `appActive`.                         |
+| overridable   | Boolean | Specifies whether the policy can be overridden by other policies.                                                                                       |
+
+**Note**: All our grantPolicies include two granting capabilities (`acknowledgeChallenge` and `pinChallenge`). If a platform does not support one of these capabilities, remove the unsupported step from the grantPolicy before execution. Using a granting capability that is not supported by the platform may result in improper behavior.
+
+### Config module Implementation
+
+To execute Usergrant test cases, additional implementation in the configuration module is required. These test cases use glue codes for mocking a grant response or a usergrant api. Below are the glue codes needed for the additional configuration.
+
+1. **Before Operation**
+   Before operations are a set of operations performed before a test case is executed. For more details, refer to the [Before Operation](https://github.com/rdkcentral/firebolt-certification-suite/blob/dev/README.md#before-operation)
+
```

Comment:
The setTestProvider function, which should be implemented in the configModule, sends the necessary messages to the platform to enable the simulation. to be changed to below

This glue code requires a custom override to be implemented in the config module in the file -(provide the path here)  setTestProvider function should be implemented in the configModule which sends necessary messages to the platform to enable the simulation.

Reply:
The heading should have the full glue code and not the substring i provided. This was just for your reference

Reply:
updated